packages/playground-examples/copy/pt/JavaScript/JavaScript Essentials/Code Flow.ts

```diff
@@ -0,0 +1,23 @@
+//// { title: "Fluxo de código", order: 3, compiler: { strictNullChecks: true } }
```

Comment:
## Playground support

The original file is "Code Flow.ts" which will override [this file](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/playground-examples/copy/en/JavaScript/JavaScript%20Essentials/Code%20Flow.ts) - in English the filename is the title, but you can change it to your own language